libraries/Microsoft.Bot.Schema/Teams/BotMeetingNotificationChannelData.cs

```diff
@@ -7,25 +7,25 @@ namespace Microsoft.Bot.Schema.Teams
     using Newtonsoft.Json;
 
     /// <summary>
-    /// Container for list of <see cref="OnBehalfOf"/> information for a meeting <see cref="TeamsMeetingNotificationChannelData"/>.
+    /// Specify Teams Bot meeting notification channel data.
     /// </summary>
-    public partial class TeamsMeetingNotificationChannelData
+    public partial class BotMeetingNotificationChannelData
```

Comment:
Out of curiosity, I see a few model classes are partial classes. Why are we doing so?

Reply:
Same question for MeetingStageSurface class.

Reply:
Good observation! 
In Eric's draft PR for meeting notification, those classes were defined as the partial class. 
Besides that, I noticed partial class is widely used in Bot SDK, including for existing Teams scenarios.

According to my research online, It has multiple advantages of using partial class instead of general class. Two main benefits I found are: 

- If you are working on an automatically generated source, then the code can be added to the class without regenerating the source file.
- If you are working on a bigger project, splitting the files over different classes helps developers work on the same project simultaneously.

I just asked what is the rule to use partial class in SDK in Bot SDK office hour group chat. Will share once get an update.

Reply:
In general, we want to avoid defining partial classes. They are usually created through auto generated code, which I believe is how the SDK was originally maintained. Let's double click with the BF team on what the right conventions are

Reply:
John and Tracy shared that partial classes are a historical way to support generated code and we can avoid it if not necessary in design.